components/Ink/src/MDCInkView.m

```diff
@@ -219,4 +245,44 @@ - (void)inkLayerAnimationDidEnd:(MDCInkLayer *)inkLayer {
   }
 }
 
+#pragma mark - CALayerDelegate
+
+- (id<CAAction>)actionForLayer:(CALayer *)layer forKey:(NSString *)event {
+  if ([event isEqualToString:@"path"] || [event isEqualToString:@"shadowPath"]) {
+
+    // We have to create a pending animation because if we are inside a UIKit animation block we
+    // won't know any properties of the animation block until it is commited.
+    MDCInkPendingAnimation *pendingAnim = [[MDCInkPendingAnimation alloc] init];
+    pendingAnim.animationSourceLayer = self.layer;
```

Comment:
In order for this to work with simulator slow animations, you would need change `self.layer` to `self.superview.layer`.  Although that only works if you are animating the bounds of the superview. I'm not sure what the ideal approach would be here.